### PR TITLE
[TASK] Move files in preparation for server separation

### PR DESCRIPTION
**Pull request by clark800**
Minimal number of changes needed to move files into new locations

### CODE REVIEW

File: api/balances.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 var Promise   = require('bluebird');
 var async     = require('async');
 var ripple    = require('ripple-lib');
-var remote    = require('./../lib/remote.js');
-var respond   = require('../lib/response-handler.js');
-var utils     = require('./../lib/utils');
-var errors    = require('./../lib/errors.js');
-var validator = require('./../lib/schema-validator.js');
+var remote    = require('./lib/remote.js');
```
Comment (lumberj) [inline api/balances.js]: Shouldn't the `api` directory be stuff for the server?

Comment (clark800) [inline api/balances.js]: No, `api` is going to move to ripple-lib. The `server` directory will be the future ripple-rest.

Comment (lumberj) [inline api/balances.js]: api has all the request handlers in ripple-rest

Comment (clark800) [inline api/balances.js]: `api` has all the business logic of ripple-rest, which is mixed with request handlers. The goal is to keep all the business logic there and extract all the request handlers to `server`.

Comment (lumberj) [inline api/balances.js]: LGTM

**Comment by geertweening** [comment]
LGTM

File: api/balances.js (hunk 1 above)
Comment (wltsmrz) [inline api/balances.js]: > No, api is going to move to ripple-lib.

Correction: /api might move to ripple-lib, but won't if I have anything to say about it

**Pull request merged**